__tests__/src/getPropValue-test.js

```diff
@@ -808,6 +808,15 @@ describe('getPropValue', () => {
 
       assert.deepEqual(expected, actual);
     });
+
+    it('should evaluate to a correct representation of an array with spread elements', () => {
+      const prop = extractProp('<div foo={[...this.props.params, bar]} />');
+
+      const expected = [undefined, 'bar'];
```

Comment:
is there no way to represent `this.props.params` here?

Reply:
@ljharb I honestly can't think of one. The values of the array would be spread in, or no values would be spread in at all. I have to return **something** that can be evaluated in code. An empty array doesn't make sense, since spreading values into an array wouldn't produce an array (it would produce more values in the containing array). The best I could come up with is `undefined`. I'm up for alternatives if you have them.

Reply:
I'm just wondering if maybe a special object indicating what was spread; that way a props-related rule could know that the "params" prop was used here (thinking of how this can be used in eslint-plugin-react)

Reply:
@ljharb I don't think we'll need it in the eslint-plugin-jsx-a11y project. Once we encounter a spread, we can't statically analyze any further.